plugin/src/commonObjects.ts

```diff
@@ -46,7 +46,7 @@ export const globals = new Set([
   '_removeShadowNodeFromRegistry',
   'RegExp',
   'Error',
-  'ErrorUtils',
+  '__ErrorUtils',
```

Comment:
Would be nice to explain why this change was necessary (as a comment, either here or in the source code).

Reply:
<img width="1083" alt="Screenshot 2023-04-05 at 14 56 22" src="https://user-images.githubusercontent.com/40713406/230087895-1c854508-b1d2-4243-836d-8a46399db517.png">
<img width="895" alt="Screenshot 2023-04-05 at 14 58 09" src="https://user-images.githubusercontent.com/40713406/230087929-1e052fc9-80fe-462d-9f9f-a0087b8270aa.png">

The greatest mystery of our time 👀
([not really](https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/Libraries/vendor/core/ErrorUtils.js#L25))